### DISCUSSION TITLE
Share of fossil fuels in the network

### DISCUSSION THREAD

**Discussion by assineu** [slipx06/sunsynk-power-flow-card#187] [Ideas]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

Can you integrate the share of fossil energy in the grid? Like the power-flow-card-plus does, for example.

### Expected behaviour

You could do this using colors, for example, like you already do with Essential.

### Mode

Both

### Context / Reason

No response

**Comment by slipx06** [comment]
Hi. On initial thought this is beyond the scope of the card which is more focused on inverter power flows and the ability to show detail of the specific inverter measurement points. The generic or higher level power flow cards would be better suited in my opinion.

**Reply by Roving-Ronin** [reply]
Would rather support for additional inverters or batteries, with the limited card space available (pending that 'wide' version ;-) ) than worrying about the fossil vs (claimed) renewable energy sources.

**Comment by Idskov** [comment]
I think this would be a great addition.
In fact I just opened #339 requesting this before I spotted that this thread was more or less the same.
